aiidalab_widgets_base/computational_resources.py

```diff
@@ -733,6 +767,11 @@ def __init__(self, **kwargs):
             style=STYLE,
         )
 
+        # Use double quotes to espace.
+        self.use_double_quotes = ipw.Checkbox(
+            value=False, description="Use double quotes"
```

Comment:
Maybe add a tooltip with a more detailed explanation of what this is used for? Also the comment in the code should explain in more detail why the escaping is needed.